ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if ! test -f "${SHARED_DIR}/sriov-worker-node"; then
+  echo "${SHARED_DIR}/sriov-worker-node file not found, no worker node for SR-IOV was deployed"
+  exit 1
+fi
+
+export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+if ! openstack network show "${OPENSTACK_SRIOV_NETWORK}" >/dev/null 2>&1; then
+    echo "Network ${OPENSTACK_SRIOV_NETWORK} doesn't exist"
+    exit 1
+fi
+NETWORK_ID=$(openstack network show "${OPENSTACK_SRIOV_NETWORK}" -f value -c id)
+
+# Re-enable the webhook on 4.9 when this PR is merged and released in 4.9z:
```

Comment:
looks like this can be enabled now

Reply:
the PR was merged but we need a new 4.9z now. Let's check again soon.

Reply:
Bugzilla already got verified https://bugzilla.redhat.com/show_bug.cgi?id=2049774

Reply:
but not released yet, will check in  a few days